src/App.tsx

```diff
@@ -1,16 +1,54 @@
+// import { AboutMe } from './components/AboutMe/AboutMe.tsx';
+import { useState } from 'react';
+
 import { AboutMe } from './components/AboutMe/AboutMe.tsx';
 import { Footer } from './components/Footer/Footer.tsx';
 import { HeaderComponent } from './components/HeaderComponent/HeaderComponent.tsx';
+import { ProductsList } from './components/ProductsList/ProductsList.tsx';
+import { PRODUCTS_DATA } from './data/data.ts';
+import type { Product } from './interfaces/Product.ts';
 
 // import styles from './App.module.css';
 
+export type Pages = 'about' | 'products';
+
 function App() {
+    const localCart = localStorage.getItem('cart') ? JSON.parse(localStorage.getItem('cart')!) : [];
```

Comment:
move 'cart' key into separate constant
```suggestion
const CartKey = "cart";
    const localCart = localStorage.getItem(CartKey) ? JSON.parse(localStorage.getItem(CartKey)!) : [];
```